wp/lib/bap_wp/src/dune

```diff
@@ -1,4 +1,4 @@
 (library
    (name bap_wp)
    (public_name bap_wp)
-   (libraries bap bap-x86-cpu z3 oUnit))
+   (libraries bap bap-x86-cpu z3 oUnit re))
```

Comment:
Should we change this to ounit2?

And also modify the `.opam` file too?

Reply:
I can definitely change this!